technika/src/App.jsx

```diff
@@ -1,11 +1,19 @@
-import { Paper, Typography } from '@mui/material';
+import { useState } from 'react';
+import { Paper } from '@mui/material';
 import { TechCard } from './techCard';
-
+import data from './data.json'
 
 function App() {
+
+  const [questions, setQuestions] = useState(data.questions)
+  const [userAnswer, setUserAnswer] = useState([])
```

Comment:
```suggestion
  const [userAnswer, setUserAnswer] = useState()
```
Turbut nedarysim keliu pasirinkimo variantu